profile/profile.yaml

```diff
@@ -0,0 +1,126 @@
+# Preferred name (Characters only)
+name: "Anthony Inthavong"
+# Your GitHub repo root URL
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_AnthonyInthavong
+
+# Introduce yourself, your academic and professional background.
+# What excites you about the software industry and your passion. At least 80 words
+summary: My name is Anthony Inthavong and I graduated from the University of Toronto in 2020, 
+        specializing in Machine Learning and Data Science with extensive experience in Programming and Algorithms. 
+        I am a versatile developer with a diverse skillset, having theoretical knowledge of computer science and 
+        machine learning, along with their practical applications. Joining Jarvis Consulting as taught me how to 
+        apply my skills in a professional working environment. Through Jarvis, I learned to apply my apply my passion as a problem solver to provide real world solutions.
+        I am very eager to meet with Jarvis' clients and not only learn from them, but also demonstrate my skill set. 
+
+# Group your skills by level.
+# Your skills can be anything software related (e.g. language, frameworks, methodologies, software, etc.).
+skills:
+  # At least 5 skills
+  proficient:
+    - Machine Learning and Data Science
+    - Data Structures and Algorithms
+    - Python
+    - Java
+    - Linux/Bash
+    - RDBMS/SQL
+    - Agile/Scrum
+    - Git
```

Comment:
reorder 
    - Java
    - Python
    - Linux/Bash
    - RDBMS/SQL
    - Agile/Scrum
    - Git
    - Machine Learning and Data Science
    - Data Structures and Algorithms